src/applications/vaos/new-appointment/components/VAFacilityPage/FacilitiesRadioWidget.jsx

```diff
@@ -24,20 +25,29 @@ export default function FacilitiesRadioWidget({
     selectedIndex >= INITIAL_FACILITY_DISPLAY_COUNT,
   );
 
+  // currently shown facility list
   const displayedOptions = displayAll
     ? enumOptions
     : enumOptions.slice(0, INITIAL_FACILITY_DISPLAY_COUNT);
-
+  // remaining facilities count
   const hiddenCount =
     enumOptions.length > INITIAL_FACILITY_DISPLAY_COUNT
       ? enumOptions.length - INITIAL_FACILITY_DISPLAY_COUNT
       : 0;
+  useEffect(
+    () => {
+      if (displayedOptions.length > INITIAL_FACILITY_DISPLAY_COUNT) {
+        scrollAndFocus(`#facility_${INITIAL_FACILITY_DISPLAY_COUNT + 1}`);
+      }
+    },
+    [displayedOptions.length, displayAll],
+  );
 
   return (
     <div>
-      {displayedOptions.map((option, i) => {
-        const { id, name, address, legacyVAR } = option?.label;
-        const checked = option.value === value;
+      {displayedOptions.map((facility, facilityIndex) => {
```

Comment:
I would probably leave the first one named `option`. It has some facility data in it, but the structure is more of an option object, which in the context of our forms means an object with a `value` and `label` property.

The second one you could leave as `i` or change to `optionIndex`.

Reply:
reverted back to `option` and `i`